### PR TITLE
Add specs for optparse into argument

### PR DESCRIPTION
**Pull request by andrykonchin**
optparse
Add an into option. [Feature #11191](https://bugs.ruby-lang.org/issues/11191)

Relates issue https://github.com/ruby/spec/issues/473

### CODE REVIEW

**Comment by andrykonchin** [comment]
@eregon There are no specs for OptionParser

File: library/optionparser/order_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'optparse'
+
+describe "OptionParser#order" do
+  ruby_version_is '2.4' do
+    it "excepts `into` keyword argument and stores result in it" do
```
Comment (eregon) [inline library/optionparser/order_spec.rb]: Did you mean `accepts`?

**Comment by eregon** [comment]
Right, the coverage of the stdlib is usually lower.
For pure-Ruby stdlib with their own tests and now their own gem/project in 2.4/2.5, I think it's less a priority to cover them than language and core.

**Comment by eregon** [comment]
Thank you for the PR!
I'll fix the small typos in the descriptions.

**Pull request merged**